example/src/runner-role.ts

```diff
@@ -55,11 +58,23 @@ export class RunnersRoleStack extends Stack {
       },
     });
 
-    new GitlabRunnerAutoscaling(this, "Runner", {
+    const runner = new GitlabRunnerAutoscaling(this, "Runner", {
       gitlabToken: gitlabToken,
       runners: {
         role: role,
       },
     });
+
+    /**
+     * Add role to runners instance profile
+     */
+    const roleForS3FullAccess = new Role(this, "RunnersRole", {
+      assumedBy: new ServicePrincipal("ec2.amazonaws.com", {}),
+      managedPolicies: [
+        ManagedPolicy.fromManagedPolicyArn(this, "AmazonS3FullAccess", "arn:aws:iam::aws:policy/AmazonS3FullAccess"),
+      ],
+    });
+
+    runner.runners.instanceProfile.roles.push(roleForS3FullAccess.roleName);
```

Comment:
Sadly, jsii doesn't allow to create methods on constructs:
```shell
addRunnersRole(role: IRole) => void;
      ~~~~~~~~~~~~~~
src/runner/runner.ts:543:3 - error JSII3001: Type "__function" cannot be used as the property type because it is private or @internal
```

So instead of using `runner.addRole(roleForS3FullAccess)` we need to go low-level